### PR TITLE
Update Pad specification

### PR DESCRIPTION
**Pull request by pelszkow**
### Details:
 - reorganize spec
 - add example - `edge` 

### Tickets:
 - 45842

### CODE REVIEW

File: docs/ops/movement/Pad_1.md [hunk 1]
```diff
@@ -57,40 +27,71 @@ depending on the *pad_mode*.
...
+
+* *pad_mode*
+
+  * **Description**: *pad_mode* specifies the method used to generate new element values.
+  * **Range of values**: Name of the method in string format:
+    * `constant` - padded values are equal to the value of the *pad_value* input, if input not provided zero value is padded.
+    * `edge` - padded values are copied from the respective edge of the input `data` tensor.
+    * `reflect` - padded values are a reflection of the input `data` tensor; values on the edges are not duplicated. `pads_begin[D]` and `pads_end[D]` must be not greater than `data.shape[D] – 1` for any valid `D`.
+    * `symmetric` - padded values are symmetrically added from the input `data` tensor. This method is similar to the `reflect`, but values on edges are duplicated. Refer to the examples above for more details. `pads_begin[D]` and `pads_end[D]` must be not greater than `data.shape[D]` for any valid `D`.
+  * **Type**: string
```
Comment (jdanieck) [inline docs/ops/movement/Pad_1.md:73]: Missing ticks around type: `string`

File: docs/ops/movement/Pad_1.md [hunk 2]
```diff
@@ -57,40 +27,71 @@ depending on the *pad_mode*.
...
+* **3**: `pads_end` - specifies the number of padding elements at the beginning of each axis. A list of non-negative integers. The length of the list must be equal to the number of dimensions in the input tensor. Required.
+
+* **4**: `pad_value` - scalar tensor of type matching type of elements in `data` tensor to be replicated in padded area. Used with the `pad_mode = "constant"` only. All new elements are populated with this value. Optional for `pad_mode = "constant"`. If input is not provided, 0 of appropriate type is used. Shouldn't be set for other `pad_mode` values.
+
+
+**Outputs**
+
+* **1**: Output padded tensor with dimensions `pads_begin[D] + data.shape[D] + pads_end[D]` for each `D` from `0` to `len(data.shape) - 1`.
+
+
```
Comment (jdanieck) [inline docs/ops/movement/Pad_1.md:92]: Missing **Types** section. Example from [ScatterNDUpdate](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/ScatterNDUpdate_3.md)

> **Types**
> T: any numeric type.

File: docs/ops/movement/Pad_1.md [hunk 3]
```diff
@@ -57,40 +27,71 @@ depending on the *pad_mode*.
...
+    * `edge` - padded values are copied from the respective edge of the input `data` tensor.
+    * `reflect` - padded values are a reflection of the input `data` tensor; values on the edges are not duplicated. `pads_begin[D]` and `pads_end[D]` must be not greater than `data.shape[D] – 1` for any valid `D`.
+    * `symmetric` - padded values are symmetrically added from the input `data` tensor. This method is similar to the `reflect`, but values on edges are duplicated. Refer to the examples above for more details. `pads_begin[D]` and `pads_end[D]` must be not greater than `data.shape[D]` for any valid `D`.
+  * **Type**: string
+  * **Default value**: None
+  * **Required**: *yes*
+
+**Inputs**
+
+* **1**: `data` - input tensor to be padded. Required.
```
Comment (jdanieck) [inline docs/ops/movement/Pad_1.md:79]: Missing information about allowed type and shape of the input tensor. Should be something like _tensor of arbitrary shape and type T_ if there are no restrictions. _T_ would be described in _Types_ section, for `data` 
probably _any numeric type_ but please double check this.

File: docs/ops/movement/Pad_1.md [hunk 4]
```diff
@@ -57,40 +27,71 @@ depending on the *pad_mode*.
...
+
+* **2**: `pads_begin` - specifies the number of padding elements at the beginning of each axis. A list of non-negative integers. The length of the list must be equal to the number of dimensions in the input tensor. Required.
+
+* **3**: `pads_end` - specifies the number of padding elements at the beginning of each axis. A list of non-negative integers. The length of the list must be equal to the number of dimensions in the input tensor. Required.
```
Comment (jdanieck) [inline docs/ops/movement/Pad_1.md:83]: Missing information about shape. To describe type please use the pattern with _T_ and _Types_ section as in other specs.

File: docs/ops/movement/Pad_1.md [hunk 5]
```diff
@@ -57,40 +27,71 @@ depending on the *pad_mode*.
...
+
+* **4**: `pad_value` - scalar tensor of type matching type of elements in `data` tensor to be replicated in padded area. Used with the `pad_mode = "constant"` only. All new elements are populated with this value. Optional for `pad_mode = "constant"`. If input is not provided, 0 of appropriate type is used. Shouldn't be set for other `pad_mode` values.
```
Comment (jdanieck) [inline docs/ops/movement/Pad_1.md:85]: I'm not sure about this, maybe @lazarevevgeny can help here. If input is optional, shouldn't we write this at the end like with _Required_?

File: docs/ops/movement/Pad_1.md [hunk 6]
```diff
@@ -57,40 +27,71 @@ depending on the *pad_mode*.
...
+
+
+**Outputs**
+
+* **1**: Output padded tensor with dimensions `pads_begin[D] + data.shape[D] + pads_end[D]` for each `D` from `0` to `len(data.shape) - 1`.
```
Comment (jdanieck) [inline docs/ops/movement/Pad_1.md:90]: Missing information about output tensor type.

File: docs/ops/movement/Pad_1.md (hunk 3 above)
Comment (pelszkow) [reply docs/ops/movement/Pad_1.md]: ok

File: docs/ops/movement/Pad_1.md (hunk 5 above)
Comment (pelszkow) [reply docs/ops/movement/Pad_1.md]: ok

File: docs/ops/movement/Pad_1.md (hunk 6 above)
Comment (pelszkow) [reply docs/ops/movement/Pad_1.md]: ok

File: docs/ops/movement/Pad_1.md (hunk 4 above)
Comment (pelszkow) [reply docs/ops/movement/Pad_1.md]: done

File: docs/ops/movement/Pad_1.md (hunk 2 above)
Comment (pelszkow) [reply docs/ops/movement/Pad_1.md:95]: added

File: docs/ops/movement/Pad_1.md (hunk 1 above)
Comment (pelszkow) [reply docs/ops/movement/Pad_1.md]: fixed

**Review by lazarevevgeny — APPROVED** [review]

**Comment by lazarevevgeny** [comment]
@pelszkow , why the PR is in "draft" state? Should we remove the "WIP" label and move to "Ready for review"?

**Review by jdanieck — APPROVED** [review]

**Comment by pelszkow** [comment]
@lazarevevgeny, it looks it might be merge.

File: docs/ops/movement/Pad_1.md [hunk 7]
```diff
@@ -122,4 +126,35 @@ OUTPUT =
...
+            <dim>4</dim>     <!-- pads_end = [1, 0, 3, 7] -->
+        </port>
+    </input>
+    <output>
+        <port id="0">
+            <dim>2</dim>     <!-- 2 = 0 + 1 + 1 = pads_begin[0] + input.shape[0] + pads_end[0] -->
+            <dim>8</dim>     <!-- 8 = 5 + 3 + 0 = pads_begin[1] + input.shape[1] + pads_end[1] -->
+            <dim>37</dim>    <!-- 37 = 2 + 32 + 3 = pads_begin[2] + input.shape[2] + pads_end[2] -->
+            <dim>48</dim>    <!-- 48 = 1 + 40 + 7 = pads_begin[3] + input.shape[3] + pads_end[3] -->
+                             <!-- all new elements are filled with 15.0 value -->
```
Comment (jane-intel) [inline docs/ops/movement/Pad_1.md:156]: Why does the elements filled with 15.0 for the edge Pad?

Comment (pelszkow) [reply docs/ops/movement/Pad_1.md:156]: It won't be fill by `15`. Fix: https://github.com/openvinotoolkit/openvino/pull/5211

**Pull request merged**